src/StaticLS/HIE/File.hs

```diff
@@ -69,11 +69,11 @@ hieFilePathToSrcFilePath = hieFilePathToSrcFilePathFromFile
 -----------------------------------------------------------------------------------
 
 -- | Retrieve an hie file from a hie filepath
-getHieFile :: (HasStaticEnv m, MonadIO m) => HieFilePath -> ExceptT HieFileReadException m GHC.HieFile
+getHieFile :: (HasCallStack, HasStaticEnv m, MonadIO m) => HieFilePath -> ExceptT HieFileReadException m GHC.HieFile
 getHieFile hieFilePath = do
     staticEnv <- getStaticEnv
-    result <- liftIO (try (GHC.readHieFile staticEnv.nameCache hieFilePath))
-    ExceptT $ pure (bimap HieFileReadException GHC.hie_file_result result)
+    result <- liftIO (fmap Right (GHC.readHieFile staticEnv.nameCache hieFilePath) `catch` (\e -> let _ = e :: SomeException in (pure . Left $ HieFileReadException)))
```

Comment:
`SomeException` is a bit aggressive here, but really I want to just discard this error ultimately in most cases